### PR TITLE
Rupato/bot 1987 update blockly to the latest version

### PR DESCRIPTION
**Pull request by rupato-deriv**

### CODE REVIEW

File: src/external/bot-skeleton/scratch/backward-compatibility.js [hunk 1]
```diff
@@ -527,7 +520,7 @@ export default class BlockConversion {
 
         this.workspace = null;
 
-        window.window.window.Blockly.Events.enable();
+        Blockly.Events.enable();
```
Comment (shafin-deriv) [inline src/external/bot-skeleton/scratch/backward-compatibility.js:523]: @rupato-deriv can we use `window.Blockly` instead?

File: src/external/bot-skeleton/scratch/backward-compatibility.js [hunk 2]
```diff
@@ -580,7 +573,7 @@ export default class BlockConversion {
                 });
             }
         } else {
-            const is_legal_block = Object.keys(window.window.window.Blockly.Blocks).includes(block_type);
+            const is_legal_block = Object.keys(Blockly.Blocks).includes(block_type);
```
Comment (shafin-deriv) [inline src/external/bot-skeleton/scratch/backward-compatibility.js:576]: same here

File: src/components/layout/header/Notifications/index.tsx [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 import { useTranslations } from '@deriv-com/translations';
-import { Notifications as UINotifications, Tooltip, useDevice } from '@deriv-com/ui';
+import { Tooltip } from '@deriv-com/ui';
 import './notifications.scss';
 
 export const Notifications = () => {
     const [isOpen, setIsOpen] = useState(false);
     const { localize } = useTranslations();
-    const { isMobile } = useDevice();
+    // TODO: need to add this back after fixing notifications component
```
Comment (sandeep-deriv) [inline src/components/layout/header/Notifications/index.tsx:10]: @rupato-deriv can we create a card for this in backlogs?

Comment (shafin-deriv) [reply src/components/layout/header/Notifications/index.tsx:10]: @sandeep-deriv Bhai I think we may not need to create any card as we were planning to handle it in the account switcher card. This part is having issues with the login for some reason which can be checked in the account switcher card.

**Review by sandeep-deriv — APPROVED** [review]

File: src/external/bot-skeleton/scratch/backward-compatibility.js (hunk 1 above)
Comment (rupato-deriv) [reply src/external/bot-skeleton/scratch/backward-compatibility.js]: done ✅

File: src/external/bot-skeleton/scratch/backward-compatibility.js (hunk 2 above)
Comment (rupato-deriv) [reply src/external/bot-skeleton/scratch/backward-compatibility.js]: done ✅

**Review by shafin-deriv — APPROVED** [review]

**Review by shafin-deriv — APPROVED** [review]

**Pull request merged**